### ISSUE TITLE
Non-useless const @var annotation

### ISSUE THREAD

**Issue by vrana** [slevomat/coding-standard#1796]
I have this code:

```php
class C {
  /** @var class-string */
  const string A = 'stdClass';
}
```

_Useless documentation comment_ is reported. It seems like an `AnnotationHelper::isAnnotationUseless` call is missing in `ClassConstantTypeHintSniff`.
